.travis.yml

```diff
@@ -71,6 +72,9 @@ before_install:
   # Remove Ruby directive from Gemfile as this image has 2.2.5
   - sed -i '/^ruby/d' Gemfile
   - mkdir -p ~/.apt-cache
+  - sudo add-apt-repository -y ppa:marutter/rdev
+  - sudo add-apt-repository -y ppa:staticfloat/juliareleases
+  - sudo add-apt-repository -y ppa:ondrej/golang
```

Comment:
Circle doesnt have a whitelist.

But these three lines are duplication, and are not whitelisted, and whitelist requests are usually very slow to be processed.

I can add a bool env var `USE_PPA_TRAVIS_TRUSTY`, so these three lines appear only once in `.ci/deps.sh`, or `.ci/deps.sh` could even use `USE_PPA` and `TRAVIS` combined to activate only these three.

I could also convert `USE_PPA` to be a string like `USE_PPA="marutter/rdev staticfloat/juliareleases ondrej/golang` in `.travis.yml`.